capsicum-test-main.cc

```diff
@@ -27,6 +30,40 @@ class SetupEnvironment : public ::testing::Environment
     }
     std::cerr << tmpdir << std::endl;
   }
+  void CheckCapsicumSupport() {
+    // For versions of googletest that lack GTEST_SKIP.
+#ifndef GTEST_SKIP
+#define GTEST_SKIP GTEST_FAIL
+#define GTEST_SKIP_defined
+#endif
```

Comment:
nit: `#define`s happening inside a function feels slightly peculiar -- could we move them to the global scope (to reflect their global impact) or maybe a header file?

Reply:
I'll move it to the global scope and not undefine it when done.

I don't want to move this to a header file because it should be a short-lived item, because (I'm predicting) the next official release of googletest will be in a couple months.

After that, according to their README.md, they won't be providing official releases. It might make more sense to switch capsicum-test to cmake and leverage the rules to pull down their repo via git and build periodically.